helm/amazon-ec2-metadata-mock/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: amazon-ec2-metadata-mock
 description: A Helm chart for the Amazon EC2 Metadata Mock
-version: 0.5.0
+version: 0.5.1
 appVersion: v0.9.3
```

Comment:
should this be updated now or after the release?

Reply:
I think the versions (aemm and chart) should be updated **before** the release, because, thinking of the source code, chart archive, the release assets should reflect the latest version and not the previous version.